go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/onsi/gomega v1.5.0
 	github.com/pkg/errors v0.8.1 // indirect
 	github.com/prometheus/client_golang v0.9.4
+	github.com/prometheus/common v0.4.1
```

Comment:
this is suspicious - hopefully the test fails

Reply:
done.